py/torch_tensorrt/dynamo/conversion/impl/linear.py

```diff
@@ -0,0 +1,85 @@
+from typing import Optional, Union
+
+import tensorrt as trt
+import torch
+from torch.fx.node import Target
+from torch_tensorrt.fx.converters.converter_utils import (
+    SourceIR,
+    get_trt_tensor,
+    set_layer_name,
+    to_numpy,
+)
+from torch_tensorrt.fx.types import TRTNetwork, TRTTensor
+
+
+def linear(
+    network: TRTNetwork,
+    target: Union[Target, str],
+    source_ir: Optional[SourceIR],
+    name: str,
+    input: TRTTensor,
+    weight: Union[TRTTensor, torch.Tensor],
```

Comment:
Switch to `Union[TRTTensor, torch.Tensor, np.ndarray],`

Reply:
Additionally, please rebase to resolve merge conflicts

Reply:
Updated! This PR relies on #2279